web/css/test.css

```diff
@@ -0,0 +1,19 @@
+[class^="test"]{
+    padding: 1rem;
+}
+
+.test-1{
+    background-color: #c00 ;
+}
+
+.test-2{
+    background-color: #036;
+}
+
+.test-3{
+    background-color: #0f0;
+}
+
+.test-4{
+    background-color: #f0f;
+}
```

Comment:
It would be nice to keep at least 1 empty line at the end of each file. Sometimes the parser (the thing that reads your code) can go horribly wrong if there is no new line.